package.json

```
@@ -177,6 +177,10 @@
     "unimported": "1.30.0",
     "zx": "^7.2.2"
   },
+  "engines": {
+    "pnpm": "9.7.0"
```

If you check the `.prototools` file, you'll see that we use `9.4.0` of `pnpm`
```suggestion
    "pnpm": "9.4.0"
```

In the past I advocated for the use of `packageManager` field which is better than engines IMO because engines is more intended for consumers of a lib not developers AFAIK.
Probably better to use `packageManager` instead of `engines` here